src/interfaces/restoration-interface.ts

```diff
@@ -0,0 +1,11 @@
+import { RestorationTypes } from '@/enums'
+
+/**
+ * A filing's restoration object from the API. See:
+ * https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/restoration.json
+ */
+export interface RestorationIF {
+  date: string // *** TODO: format?
+  type: RestorationTypes
+  expiry?: string // *** TODO: format?
+}
```

Comment:
This will need to be updated by future tickets (that implement restoration components/data) and/or schema changes.

Reply:
Also I'm not sure about the date formats yet -- probably `YYYY-MM-DDThh:mm:ss+00:00` (API format) but maybe just `YYYY-MM-DD`. To be determined by future UI/API tickets.

Reply:
Unused imports (from TodoList.vue) and TODOs are flagged as Code Smells by SonarCloud. Are we okay with adding three new smells to the code, or is there a better way to do this? (like saying "future use" instead of "TODO")

Reply:
Good catch. It's better to not have code smells.

As for TODO vs FUTURE, my style is to use TODO for things I want to complete in the current PR, and FUTURE for things that should be completed in future work. YMMV. This feature is still under development so I'm not too worried about this right now.

I've update the code. Please have a look at the latest commit.

Reply:
Sounds good!